scottypy/scotty.py

```diff
@@ -114,8 +114,7 @@ def _get_combadge(self, combadge_version):
         if self._combadge and self._combadge.version == combadge_version:
             return self._combadge
 
-        combadge_type_identifier = combadge_version if combadge_version == 'v1' else sys.platform
-        response = self._session.get("{}/combadge?combadge_version={}".format(self._url, combadge_type_identifier), timeout=_TIMEOUT)
+        response = self._session.get("{}/combadge?combadge_version={}&os_type={}".format(self._url, combadge_version, sys.platform), timeout=_TIMEOUT)
```

Comment:
If we're already touching this code, I'd suggest removal of question mark usage, and using `params` instead.